attributes/default.rb

```diff
@@ -153,8 +153,8 @@
 # default['openssh']['server']['chroot_directory'] = 'none'
 # default['openssh']['server']['banner'] = 'none'
 # default['openssh']['server']['subsystem'] = 'sftp /usr/libexec/sftp-server'
-default['openssh']['server']['trusted_user_c_a_keys'] = '/etc/ssh/ca_keys'
-default['openssh']['server']['revoked_keys'] = '/etc/ssh/revoked_keys'
+default['openssh']['server']['trusted_user_c_a_keys'] = '/etc/ssh/ca_keys' unless node['platform_family'] == 'rhel' && node['platform_version'].to_i < 6
```

Comment:
We aren't going to add this complexity to the attributes here. You could add this to your wrapper.